snap/README.md

```diff
@@ -0,0 +1,73 @@
+
+```bash
+$ sudo snap install edgex-app-service-configurable --edge
+```
+Lastly, on a system supporting it, the snap may be installed using GNOME (or Ubuntu) Software Center by searching for `edgex-app-service-configurable`.
+
+**Note** - the snap has only been tested on Ubuntu Desktop/Server versions 18.04 and 16.04, as well as Ubuntu Core versions 16 and 18.
+
+## Using the EdgeX App Service Configurable snap
+
+The App Service Configurable application service allows a variety of use cases to be met by simply providing configuration (vs. writing code). For more information about this service, please refer to the README. As with device-mqtt, this service is disabled when first installed, as it requires configuration changes before it can be run. As with the device-mqtt snap, the configuration.toml file is found in the snap’s writable area:
+
+
+/var/snap/edgex-app-service-configurable/current/config/res/
+
+### Profiles
+In additional to base configuration.toml in this directory, there are a number of sub-directories that also contain configuration.toml files. These sub-directories are referred to as profiles. The service’s default behavior is to use the configuration.toml file from the /res directory. If you want to use one of the profiles, use the snap set command to instruct the service to read its configuration from one of these sub-directories. For example, to use the push-to-core profile you would run:
+```
+$ sudo snap set edgex-app-service-configurable profile=push-to-core
+```
+In addition to instructing the service to read a different configuration file, the profile will also be used to name the service when it registers itself to the system.
+
+**Note** - as this service is based on the latest development release of EdgeX, not all use cases are supported, in particular integration with the EdgeX rules-engine will not work when used in conjunction with the Edinburgh release of EdgeX. Perform the following steps to install the edgex-app-service-configurable application service using the mqtt-export-configuration example and Mosquitto to test:
+```
+sudo snap install edgex-app-service-configurable
+
+sudo snap set edgex-app-service-configurable profile=mqtt-export
+
+sudo snap start --enable edgex-app-service-configurable.app-service-configurable
+
+mosquitto_sub -t "edgex-events"
+```
+### Multiple Instances
+Multiple instances of edgex-app-service-configurable can be installed by using snap Parallel Installs. This is an experimental snap feature and must be first be enabled by running this command:
+```
+sudo snap set system experimental.parallel-instances=true
+```
+Now you can install multiple instances of the edgex-app-service-configurable snap by specifying a unique instance name when you install the snap. The instance name is made of the name of the snap plus a unique suffix which starts with the character “_”. This name only needs to be specified for the second and susbequent instances of the snap. 
```

Comment:
note, the `_` is not part of the instance name, it is just a delimiter between the name of the snap and the instance name, so in the case of `edgexfoundry_instance`, `edgexfoundry` is the snap name and `instance` is the instance name.